fix for 'identifier "unlink" is undefined' and other similar errors

author: Kenneth Hoste (HPC-UGent)
--- clipper/sparsehash-2.0.2/src/sparsetable_unittest.cc.orig	2017-02-27 19:11:30.059750601 +0100
+++ clipper/sparsehash-2.0.2/src/sparsetable_unittest.cc	2017-02-27 19:13:28.438628098 +0100
@@ -32,7 +32,7 @@
 // Since sparsetable is templatized, it's important that we test every
 // function in every class in this file -- not just to see if it
 // works, but even if it compiles.
-
+#include <unistd.h>
 #include <sparsehash/internal/sparseconfig.h>
 #include <config.h>
 #include <stdio.h>
--- clipper/sparsehash-2.0.2/src/hashtable_test.cc.orig	2017-02-27 19:49:55.087723317 +0100
+++ clipper/sparsehash-2.0.2/src/hashtable_test.cc	2017-02-27 19:50:14.027542195 +0100
@@ -43,7 +43,7 @@
 // Note that since all these classes are templatized, it's important
 // to call every public method on the class: not just to make sure
 // they work, but to make sure they even compile.
-
+#include <stdint.h>
 #include <sparsehash/internal/sparseconfig.h>
 #include <config.h>
 #include <math.h>
--- clipper/sparsehash-2.0.2/src/time_hash_map.cc.orig	2017-02-27 20:00:27.821672176 +0100
+++ clipper/sparsehash-2.0.2/src/time_hash_map.cc	2017-02-27 20:00:57.411387012 +0100
@@ -54,6 +54,7 @@
 //
 // See PERFORMANCE for the output of one example run.
 
+#include <inttypes.h>
 #include <sparsehash/internal/sparseconfig.h>
 #include <config.h>
 #ifdef HAVE_INTTYPES_H
